### DISCUSSION TITLE
how to use specific weburl for access to webadmininterface

### DISCUSSION THREAD

**Discussion by Metanormal** [drakkan/sftpgo#1579] [Q&A]
Hej Ho *, 
i got sftpGo very easily installed and can reach the webadmin interface via default: http://www.example.com:8080/web/admin/
Since i have multiple virtual domains running on the same server, i found out that http://www.otherdomain.com:8080/web/admin/ also works, what i did not want.

So now i want to restrict access for the webadmin interface to https-protocol, a specific domain and path and port:443, like:
https://www.example.com/sftpgo/ 

I found some configurations in the sftpgo.json and am quite sure, that this is the place to do it.
But what i have to enter at what places, i dont understand. I tried the following:
``
  "httpd": {
    "bindings": [
      {
        "port": 443,
        "address": "www.example.com",
        "enable_web_admin": true,
        "enable_web_client": true,
        "enable_rest_api": true,
        "enabled_login_methods": 0,
        "enable_https": true,
        "certificate_file": "/etc/letsencrypt/live/www.example.com/fullchain.pem",
        "certificate_key_file": "/etc/letsencrypt/live/www.example.com/privkey.pem",
        "min_tls_version": 12,
        "client_auth_type": 0,
        "tls_cipher_suites": [],
        "proxy_allowed": [],

Can you help me with the missing informations? Am i going in the right direction or is the whole idea wrong? Anything?
I did not find a documentation for the sftpgo.json file and its entries, maybe you have a link to it, if it exists?

I am using apache2 under ubuntu, achieving https with letsencrypt automation.

Thanx in advnace, i say 42, Martin

**Comment by derekmahar** [comment]
Did you try setting the httpd binding `web_root` to `/sftpgo`?

**Reply by biomassboilers** [reply]
I have tried setting `web_root`, but it does not work - the login buttons redirect back to webroot/web/admin, which is a 404. SFTPGo also does not seem to embed its web code in the EXE (or at least does not serve it from there), meaning the web code must be served from the web root, which is not ideal as it conflicts with the root's index.html's dependencies. I have tried setting it to serve from a subfolder sftpgo, and adjusting sftpgo.json accordingly `"static_files_path": "/sftpgo/static/"`, but it still tries to load the web code from webroot.

**Comment by biomassboilers** [comment]
[I'm facing the same issue](https://github.com/drakkan/sftpgo/discussions/2051) - no matter what I set in web_root and in nginx config, SFTPGo doesn't serve properly.

**Comment by stratself** [comment]
If you want different webUIs on different domains, you may want to look into different bindings for each UI, and front them with different domains on the reverse proxy level. Check the final code block in #2033 as an example.
